src/shared/components/case-viewer/case-viewer.component.ts

```diff
@@ -94,11 +105,11 @@ export class CaseViewerComponent implements OnInit, OnDestroy, AfterViewInit {
     });
   }
 
-  isPrintEnabled(): boolean {
+  public isPrintEnabled(): boolean {
     return this.caseDetails.case_type.printEnabled;
   }
 
-  ngOnDestroy() {
+  public ngOnDestroy(): void {
```

Comment:
same comment here re:implemented methods public keyword

Reply:
Same as above. :)

Reply:
Reverted, pending a conversation in the Tech Forum about what we _should_ be doing as there's enormous inconsistency across within each project and across all the projects.